web-client/src/presenter/actions/JudgeActivityReport/setJudgeLastNameOnJudgeActivityReportAction.test.ts

```diff
@@ -0,0 +1,52 @@
+import { applicationContextForClient as applicationContext } from '../../../../../shared/src/business/test/createTestApplicationContext';
+import {
+  chambersUser,
+  judgeUser,
+} from '../../../../../shared/src/test/mockUsers';
+import { presenter } from '../../presenter-mock';
+import { runAction } from 'cerebral/test';
+import { setJudgeLastNameOnJudgeActivityReportAction } from './setJudgeLastNameOnJudgeActivityReportAction';
+
+describe('setJudgeLastNameOnJudgeActivityReportAction', () => {
+  presenter.providers.applicationContext = applicationContext;
+
+  it('should set state.form.judgeName to the last name of the current user when they are a judge', async () => {
```

Comment:
Does this judgeName still exist on state.form? it seems that judge activity report is no longer using state.form.

Reply:
Good catch. It'll be corrected in another branch for DOD